tomcat/src/main/scala/org/http4s/server/tomcat/TomcatBuilder.scala

```diff
@@ -136,7 +136,7 @@ sealed class TomcatBuilder[F[_]: Effect] private (
   override def start: F[Server[F]] = F.delay {
     val tomcat = new Tomcat
 
-    val context = tomcat.addContext("", getClass.getResource("/").getPath)
+    tomcat.addContext("", getClass.getResource("/").getPath)
```

Comment:
Should consider whether we want these as sequenced Effects, As `Tomcat => Unit` is a side effect for modifying tomcat. Really it is `Tomcat => Tomcat` so I'm fairly certain this constitutes an Effect.

Reply:
I still want to reform the builders so it just builds up a `Tomcat => Tomcat`.  Whether the builder steps should commute has a lot to say about the design.